R/RTemplate.R

```diff
@@ -173,9 +178,10 @@ RTscript = function() {
     )
   }
 
+  includedirs = if (opt$includedir != "") strsplit(opt$includedir,",")[[1]] else NULL
```

Comment:
I think `strsplit` returns empty vector if the input is empty string (`""`) so the `if` is not necessary.

Reply:
I still want `includedirs` to be `NULL` in case of empty input not `character(0)`

Reply:
The rest of the code should cope with both `character(0)` and `NULL`. But I'm good either way if you think the `if` makes things more clear.